### PR TITLE
add check on the hypothese automatic test status

### PR DESCRIPTION
**Pull request by ghazwarhili**

### CODE REVIEW

**Comment by ghazwarhili** [comment]
tu as une proposition pour le renommage assertNetworkAfter* ?

File: src/main/java/org/gridsuite/modification/server/dto/NetworkModificationResult.java [hunk 1]
```diff
@@ -49,4 +50,8 @@ public ApplicationStatus max(ApplicationStatus other) {
     public Set<String> getImpactedSubstationsIds() {
         return networkImpacts.stream().flatMap(impact -> impact.getSubstationIds().stream()).collect(Collectors.toCollection(TreeSet::new));
     }
+
+    public Boolean isApplicationStatusOk() {
+        return !Objects.equals(this.applicationStatus, ApplicationStatus.WITH_ERRORS);
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/modification/server/dto/NetworkModificationResult.java:55]: this.applicationStatus != ApplicationStatus.WITH_ERRORS

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

**Comment by SlimaneAmar** [comment]
Need add the test wherever we make a http request like `AbstractNetworkModificationTest` and other unit test classes

File: src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java [hunk 2]
```diff
@@ -135,14 +137,15 @@ public void testCreate() {
         ModificationInfos modificationToCreate = buildModification();
         String modificationToCreateJson = mapper.writeValueAsString(modificationToCreate);
 
-        mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
+        MvcResult mvcResult = mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn();
+        Optional<NetworkModificationResult> networkModificationResult = mapper.readValue(mvcResult.getResponse().getContentAsString(), new TypeReference<>() { });
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java:142]: Need test the status

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**